extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTask.java

```diff
@@ -499,6 +503,11 @@ public FireDepartmentMetrics getFireDepartmentMetrics()
     return fireDepartmentMetrics;
   }
 
+  private String sequenceNameForPartition(final int partitionNum)
+  {
+    return ioConfig.getBaseSequenceName() + "_" + partitionNum;
```

Comment:
minor nit, but can we use string.format here to be consistent with rest of druid code?